src/Microsoft.AspNetCore.SignalR.Core/SerializedMessage.cs

```diff
@@ -5,11 +5,27 @@
 
 namespace Microsoft.AspNetCore.SignalR
 {
+    /// <summary>
+    /// This class is designed to support the framework. The API is subject to breaking changes.
+    /// Represents a serialized message.
+    /// </summary>
```

Comment:
Same here, let's just suck it up and support this. This one's just a POVT (Plain-Old Value Type ;)) anyway.